db/models.py

```diff
@@ -1 +1,32 @@
 from django.db import models
+
+
+class Race(models.Model):
+    Race_Choices = (
+        ("ELF", "Elf"),
+        ("DWARF", "Dwarf"),
+        ("HUMAN", "Human"),
+        ("ORK", "Ork"),
+    )
+    name = models.CharField(max_length=255, unique=True, choices=Race_Choices)
+    description = models.TextField(blank=True)
+
+
+class Skill(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    bonus = models.CharField(max_length=255)
+    race = models.ForeignKey(Race, on_delete=models.CASCADE)
+
+
+class Guild(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    description = models.TextField(null=True)
+
+
+class Player(models.Model):
+    nickname = models.CharField(max_length=255, unique=True)
+    email = models.EmailField(max_length=255)
+    bio = models.CharField(max_length=255)
+    race = models.ForeignKey(Race, on_delete=models.CASCADE)
+    guild = models.ForeignKey(Guild, on_delete=models.SET_NULL, null=True)
+    create_at = models.DateTimeField(auto_now=True)
```

Comment:
From checklist:
Don't confuse `auto_now` and `auto_now_add`
- When you use `auto_now_add` - it records the time and date of creation.
- When you use `auto_now` - updates the value of the field to the current time and date every time when the object changes.

Reply:
In the task we need: `created_at - a DateTime field, that is set with the **current time** by default`, or I misunderstood?

Reply:
When you'll update a user, `created_at` value will be changed if you are using auto_now
But with auto_now_add it won't be changed
Is it clear for you now? 😊